src/lib/local-storage.js

```diff
@@ -41,7 +41,7 @@ class LocalStorage implements IStorage {
     }
 
     storage.createPackage(name, generatePackageTemplate(name), err => {
-      if (_.isNull(err) === false && err.code === STORAGE.FILE_EXIST_ERROR) {
+      if (_.isNull(err) === false && (err.code === STORAGE.FILE_EXIST_ERROR || err.code === HTTP_STATUS.CONFLICT)) {
```

Comment:
It should return 409 if the file exists already. 

The thing is, AWS, Azure or Google Cloud APIs have their own error handlings. For instance, AWS returns an error code `NotFound` or `NoSuchKey` depending on the scenario if a file is not found in the bucket. 

For the other side, Verdaccio waits for the response from the Storage Handler (either the default one, AWS, google cloud), catch it and return an HTTP error to the client package manager. In this case (perhaps is not fully documented since I've been figuring out along the time) some HTTP errors codes as 409, 500, 201, 200 have special meaning for certain actions for package managers.